tasks/jasmine.js

```diff
@@ -194,7 +195,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.showLogging) {
```

Comment:
Yeah they did already point out that they'd rather not deal with logging preference at all, but I found the existing behavior undesirable and would like some *progress* as well. 😀